faqs.toml

```diff
@@ -0,0 +1,67 @@
+# This is the FAQ file, for the FAQs
+
+# Its an array of objects with question and answer fields
+
+# Format: 
+# [[faqs]]
+# question = "Question Here"
+# answer = "Answer Here"
+# seemore = "linkhere
+
+# The seemore field is optional, and provides a "Click here to learn more" link on the FAQ.
+# It is prefaced with the host URL, such as ury.org.uk, so the example would give ury.org.uk/linkhere
```

Comment:
Hrm, wouldn't mind being able to just have a normal URL here for external links and such. (Not critical though.)